src/bit-systems/audio-target-system.ts

```diff
@@ -0,0 +1,206 @@
+import { addComponent, defineQuery, enterQuery, exitQuery } from "bitecs";
+import { Vector3, Object3D, LineSegments, WireframeGeometry, SphereBufferGeometry, PositionalAudio } from "three";
+import { HubsWorld } from "../app";
+import { AudioEmitter, AudioSettingsChanged, AudioSource, AudioTarget } from "../bit-components";
+import { SourceType } from "../components/audio-params";
+import { AUDIO_SOURCE_FLAGS } from "../inflators/audio-source";
+import { AudioSystem } from "../systems/audio-system";
+import { AudioObject3D, Emitter2Audio, EMITTER_FLAGS, makeAudioEntity } from "./audio-emitter-system";
+
+const createWhiteNoise = (audioContext: AudioContext, gain: number): AudioBufferSourceNode => {
```

Comment:
This function exists in `avatar-audio-source.js`. If you are moving it here in order to add type info, remove the function from the original file.